config/initializers/flipper.rb

```diff
@@ -0,0 +1,11 @@
+require 'flipper'
+
```

Comment:
We either need an updated Gemfile with flipper in it or this file shouldn't be here.